src/components/FeatureTrainingModal.tsx

```diff
@@ -360,6 +371,7 @@ function FeatureTrainingModal({
                       }
                     : {}),
                 ...modalInnerContainerStyle,
+                padding: 0,
```

Comment:
Why is this needed? Did we test that it does not impact the other use cases where this component is used?

Reply:
aaa yes, sorry fixing it

Reply:
@cristipaval I removed ths change